source/core/src/main/com/csse3200/game/areas/ForestGameAreaConfigs/SoundsConfig.java

```diff
@@ -4,4 +4,5 @@ public class SoundsConfig {
     public String[] gameSounds;
     public String[] gameMusic;
     public String backgroundMusic;
+    public String [] dogSounds;
```

Comment:
Unnecessary imo, but if you're going to do so then just make this animal sounds for all animals.